code/src/main/kotlin/com/expediagroup/sdk/v2/core/trait/configuration/ToClientConfigurationTrait.kt

```diff
@@ -0,0 +1,9 @@
+package com.expediagroup.sdk.v2.core.trait.configuration
+
+import com.expediagroup.sdk.v2.core.trait.Trait
+
+interface ToClientConfigurationTrait: Trait {
```

Comment:
This one looks a bit different than all other Traits, what does it mean exactly?

Reply:
UPDATE: I see it was removed in the following PRs